viz_config-example.yml

```diff
@@ -46,6 +48,12 @@ timeline_style:
   font_cex: 1
   font_col: "gray30"
 
+storm_line:
```

Comment:
this works, but wondering why `storm_line_col` isn't within the `storm_line:` list. 

Also, not really encouraging a change here, but wondering how you decide which style elements end up in the yaml vs those that end up in the function. Such as the `lty`.

Reply:
TBH, it is kind of arbitrary right now. I've been hard-coding things in the function unless I thought that I wanted to actively change it. 

And as for where this `storm_line_col` styling went ... the legend function gets passed everything under `# styling` already and this is the color that determines what the line looks like in the legend. Rather than adding an additional argument to the legend fxn, I just added it there. 

The viz_config could probably use some cleanup anyways .... if we group into `hurricane_style` rather than just have all of the prefixed `hurricane_` style cfgs, it would probably be less daunting to just add every possible style cfg, like `lty`, to the `viz_config` because it would be more organized.